styx-service-common/src/test/java/com/spotify/styx/api/ServiceAccountUsageAuthorizerTest.java

```diff
@@ -159,7 +165,8 @@ public void setUp() throws IOException {
     when(authorizationPolicy.shouldEnforceAuthorization(any(), any(), any())).thenReturn(true);
     when(idToken.getPayload()).thenReturn(idTokenPayload);
     when(idTokenPayload.getEmail()).thenReturn(PRINCIPAL_EMAIL);
-    when((Object) crm.projects().getIamPolicy(any(), any()).execute()).thenReturn(projectPolicy);
+    when((Object) getIamPolicy.execute()).thenReturn(projectPolicy);
+    when((Object) crm.projects().getIamPolicy(any(), eq(GET_IAM_POLICY_REQUEST))).thenReturn(getIamPolicy);
```

Comment:
Might be a limitation of mockito deep stub, but `when((Object) crm.projects().getIamPolicy(any(), eq(GET_IAM_POLICY_REQUEST)).execute()).thenReturn(projectPolicy);` doesn't work if later on we mock it again to get a different return value.